assembl/static2/js/app/components/debate/common/whatWeNeedToKnow.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Translate } from 'react-redux-i18n';
+
+export default ({ longTitle }) => {
+  return (
+    <div className="insert-box wwntk-box">
+      <h3 className="dark-title-3 wwntk-title">
+        <Translate value="debate.whatWeNeedToKnow" />
+      </h3>
+      <div className="box-hyphen" />
+      <p className="wwntk-text" dangerouslySetInnerHTML={{ __html: longTitle }} />
```

Comment:
Can you add a TODO comment here that we need to cleanse longTitle before setting HTML content into the `<p>` tag? This is part of our ongoing struggle with security, to not allow an attacker to override our API response and pass xss script here and we blindly add. Thanks